playbook-website/app/javascript/components/Website/src/components/DarkModeToggle.tsx

```diff
@@ -26,7 +26,7 @@ const DarkModeToggle = ({ initMode }: ToggleProps) => {
         spacing="between"
         vertical="center"
     >
-      <If condition={loading}>
+      {loading && (
```

Comment:
You will see this in a few places as well for TSX files that were JSX before. Conditional comment tags are not supported.